src/select/typeahead/_typeahead-select.scss

```diff
@@ -1,3 +1,7 @@
+.typeahead-select {
+  width: auto;
+}
```

Comment:
AFAIK, the default width value is `auto`. So, we can remove it if we are not overriding something. Wdyt?